resources/shared/helpers.mjs

```diff
@@ -23,8 +23,10 @@ export function getAllElements(selector, path = [], lookupStartNode = document)
     return elements;
 }
 
-export function forceLayout() {
-    const rect = document.body.getBoundingClientRect();
-    const e = document.elementFromPoint((rect.width / 2) | 0, (rect.height / 2) | 0);
-    return e;
+export function forceLayout(body, layoutMode = "getBoundingRectAndElementFromPoint") {
+    body ??= document.body;
+    const rect = body.getBoundingClientRect();
+    if (layoutMode === "getBoundingRectAndElementFromPoint")
+        return document.elementFromPoint((rect.width / 2) | 0, (rect.height / 2) | 0);
+    return rect.height;
```

Comment:
Should we assert that the value is `getBoundingClientRect` here?

Reply:
DONE